app/controllers/api_export_controller.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+# ApiExportController: for exporting subjects via the API
+class ApiExportController < ActionController::API
+  before_action do
+    doorkeeper_authorize!(
+      :'system/Patient.read',
+      :'system/Patient.*'
+    )
+  end
+  before_action :set_client_app
+
+  def set_client_app
+    @current_client_app = OauthApplication.find_by(id: doorkeeper_token&.application_id)
+    head :unauthorized if @current_client_app.nil?
+  end
+
+  # Multi patient PHDC export
+  def nbs_patients
+    head(:not_acceptable) && return unless request.headers['Accept']&.include?('application/zip')
+
+    search_params = params.slice('workflow', 'monitoring', 'caseStatus', 'updatedAt')
+    query = search_params.blank? ? Patient.none : Jurisdiction.find_by(id: @current_client_app[:jurisdiction_id])&.all_patients_excluding_purged
+    search_params.each do |field, search|
+      next unless search.present?
+
+      search = search.downcase.strip
+
+      case field
+      when 'workflow'
+        query = case search
+                when 'isolation'
+                  query.where(isolation: true)
+                when 'exposure'
+                  query.where(isolation: false)
+                else
+                  Patient.none
+                end
+      when 'monitoring'
+        query = case search
```

Comment:
I had only been considering that the values would come in as url encoded params. But I guess if someone wanted to instead send JSON and sent `false` instead of `"false"`, we could be smart enough to figure that out, so I will updat .

Reply:
This is handled in c971cc3.